pkg/local_object_storage/shard/mode.go

```diff
@@ -40,6 +44,10 @@ func (s *Shard) SetMode(m Mode) error {
 	s.m.Lock()
 	defer s.m.Unlock()
 
+	if s.hasWriteCache() {
+		s.writeCache.SetMode(writecache.Mode(m))
```

Comment:
`write-cache` and `shard` mode enums have to be in sync then, maybe add explicit `switch` there to prevent bugs? or use the same mode for both (if `write-cache` will always has the same modes as `shard`)

Reply:
I have thought about a test for comparison, but we also can move mode to a separate package so that all components can reuse it. Will update PR.

Reply:
Fixed.